src/yaml2candid/tests/nns-sns-wasm/SnsInitPayload.idl

```diff
@@ -1,60 +1,60 @@
 record {
```

Comment:
In the .did file, many fields below are optional.  I believe that, strictly speaking, they should therefore be `opt` even if parsers might accept either.